.github/ISSUE_TEMPLATE/oportunidade.md

```diff
@@ -78,15 +78,15 @@ Listamos abaixo alguns requisitos como um ponto de partida. Alguns são redundan
 - Conhecimentos em teoria musical
```

Comment:
Aqui pelo GitHub Mobile não consigo entender direito se essa linha foi removida. Se for o caso - acho que foi engano. Essa linha seria uma exemplo de qualificação não obrigatória mas desejável de conhecimento em teoria musical (como a vaga fictícia é relacionada com atividades musicais, a pessoa candidata teria uma vantagem competitiva na seleção se soubesse teoria musical, sacou?).

No caso, essa linha precisa ficar. É um trecho válido da oportunidade-exemplo.

Reply:
Não foi removida não, vc se confundiu 😂